# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../avifile/gcc4-h264-hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./ffmpeg/libavcodec/i386/h264dsp_mmx.c.orig	2000-01-01 00:00:00.000000000 +0100
+++ ./ffmpeg/libavcodec/i386/h264dsp_mmx.c	2005-09-14 13:57:24.000000000 +0200
@@ -260,6 +260,7 @@
 }
 static void h264_h_loop_filter_luma_mmx2(uint8_t *pix, int stride, int alpha, int beta, int8_t *tc0)
 {
+#if 0
     //FIXME: could cut some load/stores by merging transpose with filter
     // also, it only needs to transpose 6x8
     uint8_t trans[8*8];
@@ -275,6 +276,7 @@
         transpose4x4(pix-2,          trans  +2*8, stride, 8);
         transpose4x4(pix-2+4*stride, trans+4+2*8, stride, 8);
     }
+#endif
 }
 
 static inline void h264_loop_filter_chroma_mmx2(uint8_t *pix, int stride, int alpha1, int beta1, int8_t *tc0)
